README.md

```diff
@@ -1,11 +1,21 @@
 ![](./resources/official_armmbed_example_badge.png)
 # Thread statistics Mbed OS example
 
-This guide reviews the steps required to get thread statistics on an Mbed OS enabled platform.
+The example project is part of the Arm Mbed OS Official Examples and is the getting started example for Mbed OS. It contains an application that demonstrates how to generate thread statistics in Mbed OS.
```

Comment:
I don't think `and is the getting started example for Mbed OS` should be included. I'd add the same link to the examples that blinky has as well.
```suggestion
The example project is part of the [Arm Mbed OS Official Examples](https://os.mbed.com/code/). It contains an application that demonstrates how to generate thread statistics in Mbed OS.
```